src/libraries/System.Runtime/ref/System.Runtime.cs

```diff
@@ -9591,6 +9591,16 @@ public sealed partial class AssemblyTargetedPatchBandAttribute : System.Attribut
         public AssemblyTargetedPatchBandAttribute(string targetedPatchBand) { }
         public string TargetedPatchBand { get { throw null; } }
     }
+    public partial struct DependentHandle : System.IDisposable
+    {
+        private int _dummyPrimitive;
```

Comment:
It is implemented via managed reference on Mono, so this should be:
```
        private object _dummy;
        private int _dummyPrimitive;
```

Reply:
Oh, right, thanks! Fixed in https://github.com/dotnet/runtime/pull/54246/commits/ffd3e14d98450fe97dab50f74c00b3560750d7d3 😄